### PR TITLE
Ej/Enter project name view

### LINKED ISSUE #8: Migration: Migrating Project
# Migrating Project
Shows progress bar for migrating observations and medias from mapeo 5 to mapeo 6.

image

image

## Considerations
- [x] All colors should be variable in a ts file
- [x] Use [Material UI ](https://mui.com/material-ui/getting-started/overview/) for components

## To Do:
- [x] Build Screen
- [x] Add Screen to [navigation](https://github.com/digidem/mapeo-desktop-shell/issues/6)
- [x] Screen should take `observations:number` and `media:number` as props.
- [x] Increase Progress based on timer, and number of observations and media.

### PR DESCRIPTION
**Pull request by lightlii**
Adds final view to migration (_data is detected_ flow) 

closes https://github.com/digidem/mapeo-desktop-shell/issues/8

---

# Preview 
image

image

### CODE REVIEW

File: src/renderer/src/views/MigrationComplete/index.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
+import { Button, Container, FormHelperText, InputLabel, TextField, Typography, useTheme } from '@mui/material'
+import ErrorIcon from '@mui/icons-material/Error'
+import { Column, Row } from '@renderer/components/LayoutComponents'
+import { Logo } from '@renderer/components/Logo'
+import { DefaultLayout } from '@renderer/layouts/default'
+import { useState } from 'react'
+import { defineMessages, useIntl } from 'react-intl'
+import { Link } from 'react-router-dom'
```
Comment (ErikSin) [inline src/renderer/src/views/MigrationComplete/index.tsx]: Can get rid off `Link`

File: src/renderer/src/views/MigratingProject/index.tsx
Comment (ErikSin) [inline src/renderer/src/views/MigratingProject/index.tsx:1]: I can't access this file right now. Once you merge your other PR and rebase this, I can re-review

**Review by ErikSin — CHANGES_REQUESTED** [review]
5 file, new record! Thanks for keeping it scoped :)

**Comment by ErikSin** [comment]
Can you take a look at this again? There are some weird formatting thing happening, probably from the merge/rebase.

image

**Comment by lightlii** [comment]
that should be fixed now @ErikSin

**Review by ErikSin — APPROVED** [review]

**Pull request merged**